### ISSUE TITLE
Support of Redis 5+

### ISSUE THREAD

**Issue by georgasa** [eleme/corvus#149]
Hello,

are there any plans to support Redis 5.0 or higher?

Thank you,
Apostolos

**Comment by tevino** [comment]
@georgasa I'm sorry the answer is no, we don't use Corvus in eleme anymore.

**Comment by patpatbear** [comment]
@tevino  antirez was inspired by this project to anounce a new project [redis-cluster-proxy](https://github.com/artix75/redis-cluster-proxy) (owned not by antirez himself), so it's clealy an alternative approach to adopt redis-cluster with the absence of offical, stable smart client.
I'm wondering why eleme stopped Corvus development, and what's eleme using if not Covrus?

**Comment by tevino** [comment]
@patpatbear We rewrote it within the client-side sidecar for better server utilization.

**Comment by patpatbear** [comment]
is `client-side sidecar`  like [hiredis-vip](https://github.com/vipshop/hiredis-vip) , [jedis](https://github.com/xetorthio/jedis) ?

**Comment by tevino** [comment]
@patpatbear No, it's a separate process working at client-side, more like a home-brewed [envoy](https://www.envoyproxy.io/).

**Comment by patpatbear** [comment]
Interesting....
are there any more details about the `sidecar` ?
or are you planning to opensouce the `sidecar` ?

**Comment by tevino** [comment]
We are considering that, however, our sidecar sharing a lot of features with the existing envoy.

**Comment by patpatbear** [comment]
looking forward to you blogging or opensourcing the `sidecar`.

**Comment by kirk91** [comment]
It is open source now https://github.com/samaritan-proxy/samaritan,  pull requests are always welcome @patpatbear .

**Comment by patpatbear** [comment]
cheers! @kirk91
